workflow/client.go

```diff
@@ -211,3 +215,7 @@ func (c *Client) PurgeWorkflow(ctx context.Context, id string) error {
 	}
 	return c.taskHubClient.PurgeOrchestrationState(ctx, api.InstanceID(id))
 }
+
+func (c *Client) Close() {
+	_ = c.conn.Close()
```

Comment:
I think logging at the debug level here would be helpful rather than discarding any underlying errors from closing the grpc connection